cli/src/utils.js

```diff
@@ -73,6 +73,28 @@ export function stopSpinner(spinner) {
     }
 }
 
+export async function getLatestVersion() {
+    try {
+        const response = await axios.get('https://registry.npmjs.org/botfront');
+        if (response.status === 200) {
+            return response.data['dist-tags'].latest;
+        }
+        // don't do anything in case of error, just no update message
+    } catch(e) {}
+}
+
+export async function shouldUpdate() {
+    const currentVersion = getBotfrontVersion();
+    const latestVersion = await getLatestVersion();
+    return latestVersion !== currentVersion;
```

Comment:
if latestVersion is falsy, return false, and maybe display message about not being able to fetch latest version tag

Reply:
should be fixed with changes in `getLatestVersion()`